esp-hal/src/rmt.rs

```diff
@@ -684,6 +684,8 @@ mod impl_for_chip {
         }
     }
 
+    /// RMT Channel Creator
+    #[allow(missing_docs)]
```

Comment:
We probably should have _some_ documentation here

Reply:
Yea - I followed exactly what `struct Rmt` has just above. Thoughts on what it should include?

Reply:
Ah sorry .... it was too early when I looked at this, I guess.